modules/mysql/src/main/java/org/testcontainers/containers/MySQLContainer.java

```diff
@@ -56,12 +57,6 @@ public MySQLContainer(final DockerImageName dockerImageName) {
         addExposedPort(MYSQL_PORT);
     }
 
-    @NotNull
-    @Override
-    protected Set<Integer> getLivenessCheckPorts() {
-        return new HashSet<>(getMappedPort(MYSQL_PORT));
```

Comment:
This implementation in red actually a bug. The signature for this constructor is:
```
    /**
     * Constructs a new, empty set; the backing {@code HashMap} instance has
     * the specified initial capacity and default load factor (0.75).
     *
     * @param      initialCapacity   the initial capacity of the hash table
     * ...
     */
    public HashSet(int initialCapacity)
```
This implementation is actually returning an empty set.

Reply:
Oh wow, that is a nice find 😅